### PR TITLE
Condition UI description, summary

### PR DESCRIPTION
**Pull request by JoelMcKinnon**
Summary and default description fix:

For default condition changed "When all else fails" to "When no other condition is matched" and made "Define criteria" blank for default condition.

Resolves https://github.com/nasa/openmct/issues/2987

Description Any/all fix

Refactored description to do forEach through any telemetry objects used to specify criteria and set description values correctly for mixed sets of criteria.

Resolves https://github.com/nasa/openmct/issues/2988

**Author Checklist**

- Changes address original issue? Yes
- Unit tests included and/or updated with changes? Yes
- Command line build passes? Yes
- Changes have been smoke-tested? Yes

### CODE REVIEW

File: src/plugins/condition/components/Condition.vue [hunk 1]
```diff
@@ -50,11 +50,14 @@
             <span class="c-condition__name">{{ condition.configuration.name }}</span>
             <span class="c-condition__summary">
                 <template v-if="!canEvaluateCriteria">
-                    Define criteria
+                    <span v-if="!condition.isDefault">
```
Comment (charlesh88) [inline src/plugins/condition/components/Condition.vue:53]: This should use <template> rather than a span to avoid unneeded markup.

Comment (JoelMcKinnon) [reply src/plugins/condition/components/Condition.vue:53]: I don't think a template tag can have a template tag as its immediate parent. I once tried this at the top level and it didn't work. See here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template
> 

Permitted parents | Any element that accepts metadata content, phrasing content, or script-supporting elements. Also allowed as a child of a colgroup element that does not have a span attribute.
-- | --

Comment (charlesh88) [reply src/plugins/condition/components/Condition.vue]: So I just tried it in the code and it works fine. I just pushed the change back up.

Comment (JoelMcKinnon) [reply src/plugins/condition/components/Condition.vue]: In the branch @shefalijoshi merged this into she found a more elegant way to do it without the direct template/template nesting.

Comment (JoelMcKinnon) [reply src/plugins/condition/components/Condition.vue]:
```
<span class="c-condition__summary">
    <template v-if="!condition.isDefault && !canEvaluateCriteria">
        Define criteria
    </template>
<span v-else>
    <condition-description :show-label="false"
                           :condition="condition"
    />
</span>
```

**Comment by JoelMcKinnon** [comment]
Closing, as changes were merged into #2978

**Pull request closed without merging**